cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -165,11 +167,15 @@ function reconcile_addons() {
   # Filter out `configured` message to not noisily log.
   # `created`, `pruned` and errors will be logged.
   log INFO "== Reconciling with deprecated label =="
+  # shellcheck disable=SC2086
+  # Disabling because "${KUBECTL_OPTS}" needs to allow for expansion here
```

Comment:
You can use `"${KUBECTL_OPTS[@]}"` if you turn it into an array.

Reply:
IIRC this is a string all the way up as a poor way for the caller to specify additional flags :/

Reply:
Thank you for your help. @BenTheElder 
Because `${KUBECTL_OPTS}` is a string as `kubectl` argument, I chose to ignore this lint error to avoid causing other problems.

Reply:
ping @BenTheElder @thockin